rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1492,6 +1493,18 @@ public int setUnionBridgeContractAddressForTestnet(Object[] args) {
         return bridgeSupport.setUnionBridgeContractAddressForTestnet(rskTx, unionBridgeContractAddress);
     }
 
+    public long getUnionBridgeLockingCap(Object[] args) {
+        logger.trace("getUnionBridgeLockingCap");
+
+        Optional<Coin> lockingCap = bridgeSupport.getUnionBridgeLockingCap();
+
+        // This is an illegal state and should never happen
+        if (lockingCap.isEmpty()) {
+            throw new IllegalStateException("No stored or default locking cap was found.");
```

Comment:
If this is not supposed to happen, then why is it returning an Optional?

Reply:
Fixed